### DISCUSSION TITLE
Model Comparison - Understanding Model Complexity

### DISCUSSION THREAD

**Discussion by shubhambooking2** [google/meridian#168] [General]
Do we have a flexibility or a way to compute metrics such as BIC in meridian?

**Comment by alexwakim** [comment]
Information criteria such as BIC is not directly supported in Meridian. Furthermore, we don't recommend BIC when using a geo-level model. This is because the number of parameters in a hierarchical model, such as Meridian's geo-level model, isn't well defined. Something like the [Deviance Information Criteria](https://en.wikipedia.org/wiki/Deviance_information_criterion) (DIC) may make more sense. However, this is not currently directly supported. I invite you to create a feature request for this so we can assess interest.
